.github/workflows/pre-release.yml

```diff
@@ -21,10 +24,8 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: Check codestyle
-        run: pnpm lint
+      - name: Build package
+        run: pnpm -F @opennextjs/aws build
```

Comment:
this could be `pnpm build` if preferred, I set it to only build the aws package as the build script was failing in ci for some reason 😕

Reply:
Probably related to the comment i left below about pnpm